content/posts/interview_template_name_date.md

```diff
@@ -0,0 +1,24 @@
+title: Intervju med medlem: FÖRNAMN EFTERNAMN
+date: YYYY-MM-DD
```

Comment:
```suggestion
date: 2020-01-01
```
Sadly, I don't think mysgen has support for this yet, even for unpublished posts. I'll try fixing mysgen with this support later.